src/test/java/com/amplitude/api/DeviceInfoTest.java

```diff
@@ -80,15 +78,14 @@ private static Location makeLocation(String provider, double lat, double lng) {
 
     @Before
     public void setUp() throws Exception {
-        context = ShadowApplication.getInstance().getApplicationContext();
-        ShadowApplication.getInstance().getPackageManager()
-                .getPackageInfo(context.getPackageName(), 0).versionName = TEST_VERSION_NAME;
+        super.setUp(false);
+
         ReflectionHelpers.setStaticField(Build.class, "BRAND", TEST_BRAND);
         ReflectionHelpers.setStaticField(Build.class, "MANUFACTURER", TEST_MANUFACTURER);
         ReflectionHelpers.setStaticField(Build.class, "MODEL", TEST_MODEL);
 
         Configuration c = context.getResources().getConfiguration();
-        Shadows.shadowOf(c).setLocale(TEST_LOCALE);
+//        Shadows.shadowOf(c).setLocale(TEST_LOCALE);
```

Comment:
remove?

Reply:
Look like we're not using it at all. I will remove it.